src/libs/ReportActionsUtils.js

```diff
@@ -170,6 +170,28 @@ function getMostRecentIOURequestActionID(reportActions) {
     return _.last(sortedReportActions).reportActionID;
 }
 
+/**
+ * Returns array of links inside given report
+ *
+ * @param {reportAction} reportAction
+ * @returns {Boolean}
+ */
+function getLinksInsideReport(reportAction) {
+    const htmlContent = lodashGet(reportAction, ['message', 0, 'html']);
+
+    const regex = /<a\s+(?:[^>]*?\s+)?href="([^"]*)"/gi;
+
+    const urls = [];
+    let match = regex.exec(htmlContent);
+
+    while (match !== null) {
+        urls.push(match[1]);
+        match = regex.exec(htmlContent);
+    }
```

Comment:
NAB, I think maybe `string.matchAll(regex)` is a bit more obvious - but this works for sure.

Reply:
something like...

```js
return _.map([...htmlContent.matchAll(regex)], match => match[1]);
```